upgrading/topics/keycloak/changes-18_0_0.adoc

```diff
@@ -0,0 +1,20 @@
+= Step-up authentication
+
+Step-up authentication is new feature added. There is `acr` client scope, which contains protocol mapper, which is supposed to add `acr`
+claim in the token. So the `acr` claim is not added automatically now as it was before this version, but it is added with the usage
+of this client scope and protocol mapper.
+
+The client scope is added as realm "default" client scope and hence will be added to all newly created clients. But for performance reasons,
```

Comment:
It might be good that L7-L9 be one sentence.

Reply:
I keep the 3 sentences, but changed slightly the beginning of the sentences to avoid use `But` and `So` at the beginning of the sentence.

Reply:
Sorry. What I wanted to say is to remove linefeeds among L7-L9. Not write as them one sentence.

Reply:
Replied in the comment https://github.com/keycloak/keycloak-documentation/pull/1428#discussion_r824468705 . We can keep the conversation there related to line feeds.

Reply:
Yes, I agree.